src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -89,9 +89,6 @@ class LSFJobManager extends AbstractLSFJobManager {
     }
 
     private Map<BEJobID, String[]> runBjobs(List<BEJobID> jobIDs) {
-        if (!executionService.isAvailable())
```

Comment:
Currently, the only situation that this fails, is if the SSHExecutionService has connection problems. I think this is an exceptional situation, actually -- it should indeed be fatal, because it means that the remote command cannot be executed!

Therefore, a clean interface would be to throw an exception, if the connection is not established (after maybe retrying -- however, that would be the responsibility of the service itself, not here). I am not sure whether throwing itself is the responsibility of the service or the job manager, though.

@dankwart-de BTW: Calling an execution service with an empty string currently results in a empty result containing some error message. I am not convinced that this is a good interface. In my opinion, an empty string argument there is always a programming error and should throw a probably unchecked exception.